docs/intro.rst

```diff
@@ -89,23 +89,25 @@ through auto-detection as the default configuration, contains the following
 profiles::
 
           [defaults]
-          endpoint = https://url.of.some.dwavesystem.com/sapi
-          client = qpu
-
-          [dw2000]
-          solver = DW_2000Q_1
           token = ABC-123456789123456789123456789
 
+          [firstAvailableQPU]
```

Comment:
I would avoid the lower camel case (non-standard Python) and use `first-available-qpu`.

Reply:
I'll make all these changes